pkg/services/sqlstore/migrations/ualert/cond_trans.go

```diff
@@ -187,7 +187,7 @@ func transConditions(set dashAlertSettings, orgID int64, dsUIDMap dsUIDLookup) (
 	ccAlertQuery := alertQuery{
 		RefID:         ccRefID,
 		Model:         exprModelJSON,
-		DatasourceUID: "-100",
+		DatasourceUID: "__expr__",
```

Comment:
rather than using the magic string "__expr__" can we import the go constant?

This whole mess was caused by using the magic string "-100" as a uid... so we still need to make sure that continues to work when it is saved in the raw JSON models

Reply:
Yes, we can and should. I will handle that part.